### PR TITLE
Revert "Merge pull request #3076 from void-ghost/stereo3d_presets"

### PR DESCRIPTION
**Pull request by Armada651**
The code is unnecessarily boilerplate and without being able to save presets it's not very useful. I'd like to revert this PR until a better implementation comes around.

### CODE REVIEW

**Comment by degasus** [comment]
May you explain more explicit why you want to revert this PR?
Maybe:
- Why don't you want this feature?
- In which better way do you think this should be done?
- How might the code be improved to be fine?

Maybe @void-ghost come up with a different solution.

**Comment by Armada651** [comment]
- The values needed for convergence are completely specific to a certain game. Instead of using presets the user should be encouraged to set these values in the game-specific properties instead.
- This should be done in the GameINI, where it is already handled.
- For one thing you should actually be able to save these presets from the user interface, but that still calls into question what this feature would actually solve that isn't handled much better by setting game-specific configurations?

File: Source/Core/DolphinWX/Frame.cpp [hunk 1]
```diff
@@ -1482,72 +1482,29 @@ void CFrame::ParseHotkeys()
...
 		g_Config.iStereoConvergence += 5;
 		if (g_Config.iStereoConvergence > 500)
 			g_Config.iStereoConvergence = 500;
-		g_Config.oStereoPresets[g_Config.iStereoActivePreset].convergence = g_Config.iStereoConvergence;
-		savePreset("Convergence", g_Config.iStereoConvergence);
 	}
 
-	if (IsHotkey(HK_SWITCH_STEREOSCOPY_PRESET))
-	{
-		g_Config.iStereoActivePreset = !g_Config.iStereoActivePreset;
```
Comment (Armada651) [inline Source/Core/DolphinWX/Frame.cpp]: This line will only switch between preset 0 and 1 using a boolean operator...

File: Source/Core/DolphinWX/Frame.cpp [hunk 2]
```diff
@@ -1482,72 +1482,29 @@ void CFrame::ParseHotkeys()
...
-		savePreset("Convergence", g_Config.iStereoConvergence);
 	}
-	if (IsHotkey(HK_INCREASE_CONVERGENCE))
+	if (IsHotkey(HK_INCREASE_CONVERGENCE, true))
 	{
 		g_Config.iStereoConvergence += 5;
 		if (g_Config.iStereoConvergence > 500)
 			g_Config.iStereoConvergence = 500;
-		g_Config.oStereoPresets[g_Config.iStereoActivePreset].convergence = g_Config.iStereoConvergence;
-		savePreset("Convergence", g_Config.iStereoConvergence);
```
Comment (Armada651) [inline Source/Core/DolphinWX/Frame.cpp]: Don't save settings outside of `VideoConfig::Save`.

File: Source/Core/DolphinWX/Frame.cpp [hunk 3]
```diff
@@ -1482,72 +1482,29 @@ void CFrame::ParseHotkeys()
...
-	auto savePreset = [](const std::string& param, int value)
-	{
-		IniFile localIni = SConfig::GetInstance().LoadLocalGameIni();
-		localIni.GetOrCreateSection("Enhancements")->Set(
-			StringFromFormat("Stereo%s_%d", param.c_str(), g_Config.iStereoActivePreset),
-			value);
-		std::string iniFileName = File::GetUserPath(D_GAMESETTINGS_IDX) + SConfig::GetInstance().GetUniqueID() + ".ini";
-		OSD::AddMessage(StringFromFormat("%s: %d", param.c_str(), value) , 1000);
-		localIni.Save(iniFileName);
-	};
```
Comment (Armada651) [inline Source/Core/DolphinWX/Frame.cpp]: Don't save settings outside of `VideoConfig::Save`.

File: Source/Core/VideoCommon/VideoConfig.cpp [hunk 4]
```diff
@@ -98,16 +98,6 @@ void VideoConfig::Load(const std::string& ini_file)
...
 
-	//currently these settings are not saved in global config, so we could've initialized them directly
-	for (size_t i = 0; i < oStereoPresets.size(); ++i)
-	{
-		stereoscopy->Get(StringFromFormat("StereoConvergencePercentage_%zu", i), &oStereoPresets[i].convergence, iStereoConvergencePercentage);
-		stereoscopy->Get(StringFromFormat("StereoDepth_%zu", i), &oStereoPresets[i].depth, iStereoDepth);
-	}
-	stereoscopy->Get("StereoActivePreset", &iStereoActivePreset, 0);
-	iStereoConvergencePercentage = oStereoPresets[iStereoActivePreset].convergence;
-	iStereoDepth = oStereoPresets[iStereoActivePreset].depth;
```
Comment (Armada651) [inline Source/Core/VideoCommon/VideoConfig.cpp]: These settings should not be applied globally.

**Pull request merged**